examples_failing/pedersen/src/main.nr

```diff
@@ -1,10 +0,0 @@
-// This is the Pedersen hash function.
```

Comment:
Why does this PR remove this and the other examples?

Reply:
It was moved to the `test_data` folder as they were passing examples, and not failing so they are mislabeled. I could make a separate PR quickly, but I just did it in here as I was fixing up my own simple_shield example.